pysat/tests/test_files.py

```diff
@@ -698,14 +863,15 @@ def test_get_new_files_after_adding_files_and_adding_file(self):
         new_files2 = self.testInst.files.get_new()
         assert np.all(new_files.index == dates)
         assert np.all(new_files2.index == dates2)
+        return
 
     def test_get_new_files_after_deleting_files_and_adding_files(self):
-        """Check that get_new works after deleting and adding files"""
-        # create new files and make sure that new files are captured
+        """Check that get_new works after deleting and adding files."""
+        # Create new files and make sure that new files are captured
         dates = pysat.utils.time.create_date_range(self.start2, self.stop2,
                                                    freq='100min')
 
-        # remove files, same number as will be added
+        # Remove files, same number as will be added
         to_be_removed = len(dates)
         for the_file in os.listdir(self.testInst.files.data_path):
             if (the_file[0:13] == 'pysat_testing') & \
```

Comment:
Move operator after break and use `and` instead of `&`.  Also don't need the `(` `)` for the separate statements.